firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -434,6 +435,16 @@ private static ConfigContainer extractConfigs(JSONObject fetchResponse, Date fet
         containerBuilder.withPersonalizationMetadata(personalizationMetadata);
       }
 
+      String templateVersionNumber = null;
+      try {
+        templateVersionNumber = fetchResponse.getString(TEMPLATE_VERSION_NUMBER);
```

Comment:
Maybe `optString` https://developer.android.com/reference/org/json/JSONObject#optString(java.lang.String,%20java.lang.String)

Reply:
Just following the convention that was left for the other response objects.